app/konnect/servicehub/manage-services.md

```diff
@@ -1,80 +1,69 @@
 ---
-title: Manage Konnect Services
+title: Konnect Service Dashboard
```

Comment:
I think this doc might need to be deleted since you combined the overview and dashboard. Right now it's unlisted but will show up in the Konnect docs.